lib/cell.rb

```diff
@@ -0,0 +1,46 @@
+require './lib/ship'
+
+class Cell
+  attr_reader :coordinate, :ship, :fire_upon
+
+  def initialize(coordinate)
+    @ship = nil
+    @coordinate = coordinate
+    @fired_upon = false
+    @render = "."
+  end
+
+  def empty?
+    if self.ship == nil
```

Comment:
After pulling down I did try just @ship == nil here and it still passed! I think the error might have been from earlier. Might be cleaner to change all these to @ship instead of self.ship now that we got that ironed out.